libraries/rush-lib/src/utilities/npmrcUtilities.ts

```diff
@@ -22,13 +22,25 @@ export interface ILogger {
 // create a global _combinedNpmrc for cache purpose
 const _combinedNpmrcMap: Map<string, string> = new Map();
 
-function _trimNpmrcFile(sourceNpmrcPath: string, extraLines: string[] = []): string {
+function _trimNpmrcFile(
+  sourceNpmrcPath: string,
+  extraLines: {
+    lines: string[];
+    insertBefore: boolean;
+  } = { lines: [], insertBefore: false }
+): string {
```

Comment:
This function signature is getting messy.  Maybe we can do something like:

```ts
function _trimNpmrcFile(options: {
  sourceNpmrcPath: string,
  linesToPrepend?: string[],
  linesToAppend?: string[],
): string {
```

Reply:
Changed to use an option object